### DISCUSSION TITLE
Thermostat driver

### DISCUSSION THREAD

**Discussion by MrNomNom** [arendst/Tasmota#13940] [Show and tell]
Hi all, started using Sonoff 4ch relay with 3 heating zones. So far I am really happy how stable heating is (I am using basically default settings). 

Thanks devs ;) 

Ps. my temp sensors are Aqara Temperature and Humidity sensors.

Adding screenshot of one heating zone, really cool stuff :)
![Screenshot_2021-12-07T094432](https://user-images.githubusercontent.com/1655555/144990631-636357c3-9982-430f-82b6-26d15acafbf9.png)

Here is how wiring looks like (not the prettiest one :)
![IMG_9160](https://user-images.githubusercontent.com/1655555/144990968-ad9095ca-cf19-4e75-b5ff-254ae9ba8809.jpg)

**Comment by rapi3** [comment]
Tasmota project and community it is really amazing !

I just discovered Tasmota project this year and I made: Alarm system, sensors and now a thermostat with Tasmota firmware.
It is really important for me because I can control stuff in the house when I am not at home for a long period which is 1/2 of the year.

I just switched from rpi I used to control the temp in the house for last 5 years with one RF 433 relay and I made my own thermostat with a Sonoff 1ch modified and firmware only with DS18B20 sensor.
All I miss at the moment from the old solution it is the possibility to quick adjust the desired temp from a web interface but I think I can solve this in the future, I just need to send by mqtt new value for mem1-8 variable I use.

It monitor the rooms temp from 8 external sensors I made with wemos D1 mini + DS18B20 and it control the main heating.
The heating It is also stable because the intervals I set are independent for each room - and depend also of sensors location in the room from heating radiators.
At this moment I use +0.2C and +0.5C range to start-stop the heating depend of room.

![Screenshot_2021-12-08_14-07-41](https://user-images.githubusercontent.com/18531150/145215284-61631023-6c15-4be3-8487-3a283520849e.png)
![Screenshot_2021-12-08_14-08-10](https://user-images.githubusercontent.com/18531150/145215317-e03bf503-aa06-4d57-a58f-a1848e24a1a6.png)

![Screenshot_2021-12-08_14-00-31](https://user-images.githubusercontent.com/18531150/145213248-7fd636d5-4d1e-48ec-86cd-90e55baf8656.png)
![Screenshot_2021-12-08_14-00-49](https://user-images.githubusercontent.com/18531150/145213348-3a468153-e5c3-4795-a542-829206dde436.png)
![Screenshot_2021-12-08_13-51-07](https://user-images.githubusercontent.com/18531150/145213299-a9872218-143c-4d4d-ad1f-c1b9941dbcbb.png)
![Screenshot_2021-12-08_14-01-23](https://user-images.githubusercontent.com/18531150/145213413-e05420a4-007b-43cd-a08e-6b1342e7c110.png)
![Screenshot_2021-12-08_14-01-40](https://user-images.githubusercontent.com/18531150/145213421-a8aff325-dda0-40b2-b0da-1e70e94bdca7.png)
